test/features/valid.csv

```diff
@@ -0,0 +1,301 @@
+John Smith,A1234BY,1986-12-09,INP
+John Smith,A1234BY,1986-12-09,INP
```

Comment:
Originally yes, as I used it for testing the spinny wheel and it had to be long enough.